nums/numpy/api.py

```diff
@@ -399,6 +399,44 @@ def any(a: BlockArray, axis=None, out=None, keepdims=False):
         raise NotImplementedError("'out' is currently not supported.")
     return _instance().reduce("any", a, axis=axis, keepdims=keepdims)
 
+############################################
+# NaN Ops
+############################################
+
+
+def nanmax(a: BlockArray, axis=None, out=None, keepdims=False):
+    if out is not None:
+        raise NotImplementedError("'out' is currently not supported.")
+    return _instance().reduce("nanmax", a, axis=axis, keepdims=keepdims)
+
+
+# def nanmedian(a: BlockArray, axis=None, out=None, overwrite_input=False, keepdims=False):
```

Comment:
Could we remove the commented code? You could tag / branch before removing to refer to later.

Reply:
Yes.